tests/test_jmcomic/test_jm_async_client.py

```
@@ -0,0 +1,264 @@
+
+        for expected_id, params in cases.items():
+            try:
+                sync_page = self.sync_api_client.search_site(**params)
+                async_page = self.run_async(self.async_client.search_site(**params))
+                sync_first_aid = int(sync_page[0][0])
+                async_first_aid = int(async_page[0][0])
+                self.assert_sync_async_equal(sync_first_aid, async_first_aid,
+                                             f'search_params[{expected_id}].first_aid')
+            except Exception:
+                # 与 sync 原测试一致：网络/排名波动时不阻塞
+                pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)